extensions/wikia/CookieSyncer/templates/CookieSyncer_Index.php

```diff
@@ -1 +1,9 @@
-<iframe src="<?= $url; ?>" class="cookie-syncer-module-iframe"></iframe>
+<script type="text/javascript">
+	if (window.document.cookie.indexOf('cookiesync_done') === -1 && window.document.cookie.indexOf('tracking-opt-in-status') !== -1) {
```

Comment:
If we use AssetsManager to load this script, then we can use https://github.com/Wikia/app/blob/dev/resources/wikia/modules/cookies.js#L58 instead of `indexOf` which might break in some edge cases.

Reply:
fix in 44a754fb8d8